packages/dev/docs/src/Layout.js

```diff
@@ -115,37 +129,105 @@ export function Layout({scripts, styles, pages, currentPage, publicUrl, children
         {scripts.map(s => <script type={s.type} src={s.url} defer />)}
       </head>
       <body>
-        <div className={docStyles.pageHeader} id="header" />
-        <nav className={docStyles.nav}>
-          <header>
-            <a href={publicUrl}>
-              <svg viewBox="0 0 30 26" fill="#E1251B">
-                <polygon points="19,0 30,0 30,26" />
-                <polygon points="11.1,0 0,0 0,26" />
-                <polygon points="15,9.6 22.1,26 17.5,26 15.4,20.8 10.2,20.8" />
-              </svg>
-              <h2 className={typographyStyles['spectrum-Heading4']}>React Spectrum</h2>
-            </a>
-          </header>
-          <ul className={sideNavStyles['spectrum-SideNav']}>
-            {pages.filter(p => p.name !== 'index.html' && (currentPage === 'index.html' || p.name.split('/')[0] === currentPage.split('/')[0])).map(p => (
-              <li className={classNames(sideNavStyles['spectrum-SideNav-item'], {[sideNavStyles['is-selected']]: p.name === currentPage})}>
-                <a className={sideNavStyles['spectrum-SideNav-itemLink']} href={p.url}>{path.basename(p.name, path.extname(p.name))}</a>
-              </li>
-            ))}
-          </ul>
-        </nav>
-        <main>
-          <article className={typographyStyles['spectrum-Typography']}>
-            <MDXProvider components={mdxComponents}>
-              <LinkProvider>
-                {children}
-              </LinkProvider>
-            </MDXProvider>
-          </article>
-          <ToC toc={toc} />
-        </main>
+        {children}
       </body>
     </html>
   );
 }
+
+function dirToTitle(dir) {
+  return dir
+    .split('/')[0]
+    .split('-')
+    .map(w => w.charAt(0).toUpperCase() + w.slice(1))
+    .join(' ');
+}
+
+function Nav({currentPageName, pages, publicUrl}) {
+  let isIndex = /index\.html$/;
+  let currentParts = currentPageName.split('/');
+  let currentDir = currentParts[0];
+    
+  pages = pages.filter(p => {
+    let pageParts = p.name.split('/');
+    let pageDir = pageParts[0];
+
+    // Pages within same directory (react-spectrum/Alert.html)
+    if (currentParts.length > 1) {
+      return currentDir === pageDir && !isIndex.test(p.name);
+    }
+
+    // Top-level index pages (react-spectrum/index.html)
+    if (currentParts.length === 1 && pageParts.length > 1 && isIndex.test(p.name)) {
+      return true;
+    }
+
+    // Other top-level pages
+    return !isIndex.test(p.name) && pageParts.length === 1;
+  });
+
+  let title = currentParts.length > 1 ? dirToTitle(currentPageName) : 'React Spectrum';
```

Comment:
this is a bit of a hack but mostly works for now. basically, turn the sub directory into the title (eg `react-spectrum` => `React Spectrum`). happy for any other ideas